core/src/test/java/com/linecorp/armeria/internal/annotation/AnnotatedHttpDocServiceTest.java

```diff
@@ -88,6 +92,7 @@
     public static final ServerRule server = new ServerRule() {
         @Override
         protected void configure(ServerBuilder sb) throws Exception {
+            //sb.http(8080);
```

Comment:
How about adding a static final boolean field like: `boolean DEMO_MODE = false` and calling `sb.http(8080)` when it's `true`?